less/pages/healthcheck.less

```diff
@@ -0,0 +1,16 @@
+.page.healthcheck {
+  .sidebar,
+  footer {
+    display: none !important;
+  }
+  main {
+    width: 100%;
+  }
```

Comment:
```css
.sidebar,
  footer {
    display: none !important;
  }
  main {
    width: 100%;
  }
```
is just a lazy way to implement [option 2](https://github.com/mozilla/teach.webmaker.org/pull/1072#issue-92952486) design.  :eyes: